web/sites/default/config/default/webform.webform.edit_your_cssn_roles.yml

```diff
@@ -23,13 +23,13 @@ elements: |-
     '#type': checkboxes
     '#title': 'Roles:'
     '#options':
-      Mentor: 'Mentor -- Provides or is willing to provide guidance, advice, feedback and support to a mentee. '
-      Student: |
-        Student -- A person enrolled in an institution of higher learning
-      'Research Computing Facilitator': 'Research Computing Facilitator -- Assists researchers with accessing data, study design, statistical and analytical methods, and coordinates the use of computing services, software licensing, and associated technical support for new and ongoing research projects and academic endeavors.'
-      'Research Software Engineer': 'Research Software Engineer -- Works with researchers to gain an understanding of the problems they face, and then develops, maintains and extends software to provide the answer'
-      'CI Systems Engineer': 'CI Systems Engineer -- Provides direct centralized support on scientific computing to campus researchers to enhance computational thinking and modern computational methods'
-      'Researcher / Educator': 'Researcher / Educator -- A person doing research or teaching at an institution of higher learning'
+      mentor: 'mentor -- Provides or is willing to provide guidance, advice, feedback and support to a mentee. '
+      student: |
+        student -- A person enrolled in an institution of higher learning
+      research_computing_facilitator: 'research_computing_facilitator -- Assists researchers with accessing data, study design, statistical and analytical methods, and coordinates the use of computing services, software licensing, and associated technical support for new and ongoing research projects and academic endeavors.'
+      research_software_engineer: 'research_software_engineer -- Works with researchers to gain an understanding of the problems they face, and then develops, maintains and extends software to provide the answer'
+      ci_systems_engineer: 'ci_systems_engineer -- Provides direct centralized support on scientific computing to campus researchers to enhance computational thinking and modern computational methods'
+      researcher: 'researcher -- A person doing research or teaching at an institution of higher learning'
```

Comment:
![image](https://user-images.githubusercontent.com/539763/233435228-f1ca9df2-1557-4091-aa06-dfbf1f9f4ca6.png)
The Community Persona page is displaying "friendly names" rather than the machine names. Probably we should do that here also.

Reply:
I only meant to change the option names not the 'friendly' name. Just pushed a change to get those back inline to display the  'friendly' names to the user.